examples/main/utils.js

```diff
@@ -25,3 +26,80 @@ export function pointGrid(N, bbox) {
 
   return points;
 }
+
+export function pointsToWorldGrid(points, worldUnitSize) {
```

Comment:
This could be in its own file named e.g. `grid-aggregator.js`. We can then also provide a `hexagon-aggregator.js` with a similar interface for that example.